app/models/congress_message.rb

```diff
@@ -69,8 +69,7 @@ def background_submit(test: false)
     return unless valid?
 
     @forms.each do |f|
-      f.delay.fill(attributes_for(f.bioguide_id), campaign.campaign_tag,
-                   test: test)
+      f.delay.fill(attributes_for(f.bioguide_id), campaign.campaign_tag, test)
```

Comment:
Looks like rubocop 'fixed' this here: https://github.com/EFForg/action-center-platform/commit/9ecfc7d2fcd031ca70c546acc23c36a60ab355b6

Reply:
I need to add something so rubocop doesnt make the same mistake again

Reply:
I got rubocop running and it did not try to fix this again so I think its good to go!